Common/Scheduling/DateRules.cs

```diff
@@ -212,6 +270,12 @@ public IDateRule WeekEnd(Symbol symbol)
         /// <returns>The security object matching the given symbol</returns>
         private Security GetSecurity(Symbol symbol)
```

Comment:
It's almost as this method should actually return the `SecurityExchangeHours` instance directly, would save us from duplicating the `var securitySchedule = security == null ? SecurityExchangeHours.AlwaysOpen(TimeZones.NewYork) : security.Exchange.Hours;` and passing around a security instance